captum/optim/_param/image/images.py

```diff
@@ -137,13 +149,29 @@ def __init__(
         self.fourier_coeffs = nn.Parameter(fourier_coeffs)
 
     def rfft2d_freqs(self, height: int, width: int) -> torch.Tensor:
-        """Computes 2D spectrum frequencies."""
+        """
+        Computes 2D spectrum frequencies.
+
+        Args:
+            height (int): The h dimension of the 2d frequency scale.
+            width (int): The w dimension of the 2d frequency scale.
+        Returns:
+            tensor (tensor): A 2d frequency scale tensor.
+        """
+
         fy = self.torch_fftfreq(height)[:, None]
         fx = self.torch_fftfreq(width)[: width // 2 + 1]
         return torch.sqrt((fx * fx) + (fy * fy))
 
     def get_fft_funcs(self) -> Tuple[Callable, Callable, Callable]:
-        """Support older versions of PyTorch"""
+        """
+        Support older versions of PyTorch.
```

Comment:
nit: Do you mind adding more documentation to this method ? It does more than supporting older PyTorch version, doesn't it ?

Reply:
It just ensures that the same FFT operations are carried regardless of whether your PyTorch version has the `torch.fft` update.